test/rendering-test/test-rendering.js

```diff
@@ -155,7 +155,11 @@ class RenderingTest extends Component {
       height: height,
       debug: true,
       onAfterRender: this._onDrawComplete.bind(this, name, referecenResult)
-    }, mapViewState, {
+    },
+    {
+      viewports: [new WebMercatorViewport(viewportProps)]
```

Comment:
Why use `_extends` instead of a plain object?

Reply:
Also just use `viewport`. `viewports` is still experimental.

Reply:
fixed